src/main/java/uk/gov/pay/products/validations/RequestValidations.java

```diff
@@ -93,6 +98,10 @@ private static Function<JsonNode, Boolean> isNotNumeric() {
         return jsonNode -> !isDigits(jsonNode.asText());
     }
 
+    private static Function<JsonNode, Boolean> isNotGreaterThanZero() {
+        return jsonNode -> isDigits(jsonNode.asText()) && jsonNode.asLong() == ZERO_PRICE;
```

Comment:
Honestly, I’d just put a literal `0L` here and not have the constant.

Reply:
Done